tuxemon/states/combat/combat.py

```diff
@@ -977,6 +981,10 @@ def perform_action(
                     # TODO: Don't end combat right away; only works with SP,
                     # and 1 member parties end combat right here
                     if result["success"]:
+                        # Tuxepedia: set monster as caught (2)
+                        self.players[0].tuxepedia[
+                            self.monsters_in_play[self.players[1]][0].slug
```

Comment:
can we just use `target` rather than `self.monsters_in_play[self.players[1]][0].slug`  ?

Reply:
ok